docs/deployment-guide.md

```
@@ -400,52 +311,35 @@ curl -X POST https://your-worker.workers.dev/events \
+|----------|-------------|
+| `DESTINATION_TYPE` | Set to `GrafanaCloud` |
+| `GRAFANA_CLOUD_INSTANCE_ID` | Your Instance ID |
+| `GRAFANA_CLOUD_API_KEY` | Your Cloud API Key |
 
 ### Setting Secrets
 
-**For Cloudflare Workers:**
+The Go worker reads secrets from environment variables or a `.env` file in the root of the `worker/` directory.
+
+**Environment Variable:**
 ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)